adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -76,21 +95,45 @@ def connect(self):
             for access_pt in self._esp.scan_networks():
                 print("\t%s\t\tRSSI: %d" % (str(access_pt['ssid'], 'utf-8'), access_pt['rssi']))
         failure_count = 0
-        while not self._esp.is_connected:
-            try:
-                if self.debug:
-                    print("Connecting to AP...")
-                self.pixel_status((100, 0, 0))
-                self._esp.connect_AP(bytes(self.ssid, 'utf-8'), bytes(self.password, 'utf-8'))
-                failure_count = 0
-                self.pixel_status((0, 100, 0))
-            except (ValueError, RuntimeError) as error:
-                print("Failed to connect, retrying\n", error)
-                failure_count += 1
-                if failure_count >= self.attempts:
+        if self.wificonntype == WiFiConnType.normal:
+            while not self._esp.is_connected:
+                try:
+                    if self.debug:
+                        print("Connecting to AP...")
+                    self.pixel_status((100, 0, 0))
+                    self._esp.connect_AP(bytes(self.ssid, 'utf-8'), bytes(self.password, 'utf-8'))
+                    failure_count = 0
+                    self.pixel_status((0, 100, 0))
+                except (ValueError, RuntimeError) as error:
+                    print("Failed to connect, retrying\n", error)
+                    failure_count += 1
+                    if failure_count >= self.attempts:
+                        failure_count = 0
+                        self.reset()
+                    continue
+        elif self.wificonntype == WiFiConnType.enterprise:
+            self._esp.wifi_set_network(bytes(self.ent_ssid, 'utf-8'))
```

Comment:
change to `ssid`

Reply:
This is being discussed in another comment/suggestion. Resolving here and will address as needed.